test/models/wishlist_item_test.rb

```diff
@@ -72,4 +72,31 @@ def test_domain_name_must_be_a_valid_domain
     assert_not(item.valid?)
     assert_equal(['is invalid'], item.errors[:domain_name])
   end
+
+  def test_domain_has_valid_extension
+    supported_extensions = ["ee", "pri.ee", "com.ee", "med.ee", "fie.ee"]
```

Comment:
Do they all really needed here? What if we create a single one and leave only two tests - `valid` and `invalid`?